### ISSUE TITLE
wsl : The term 'wsl' is not recognized

### ISSUE THREAD

**Issue by SeanBatt** [MicrosoftDocs/WSL#725]
I'm not even a little bit surprised following the instructions leads to this failure. Not surprised to see many others have the same issue. I mean, IT is hard! One should expect to spend hours doing simple tasks and to have to do lots of research into what the official documents really meant to say. If one is not prepared for that, well! Might as well use Apple gear!
---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: b7a1833a-3461-2d9e-bb1e-de5d0ee3fe33
* Version Independent ID: bd4b2627-60c7-1bfe-7831-1dbde906e287
* Content: [Install Windows Subsystem for Linux (WSL) on Windows 10](https://docs.microsoft.com/en-us/windows/wsl/install-win10)
* Content Source: [WSL/install-win10.md](https://github.com/MicrosoftDocs/wsl/blob/master/WSL/install-win10.md)
* Product: **dev-environment**
* Technology: **windows-subsystem-for-linux**
* GitHub Login: @craigloewen-msft
* Microsoft Alias: **crloewen**

**Comment by craigloewen-msft** [comment]
Hey @SeanBatt you're right our install instructions are pretty long! We've committed to [making this better](https://devblogs.microsoft.com/commandline/the-windows-subsystem-for-linux-build-2020-summary/#wsl-install). 

For your problem it sounds like the Windows Subsystem for Linux optional component isn't enabled, can you double check that it is by running
`dism.exe /online /enable-feature /featurename:Microsoft-Windows-Subsystem-Linux /all /norestart`

And please let me know if there's any other way we can improve the docs to make them clearer.

**Comment by mattwojo** [comment]
Closing this issue. Please reopen if there's something else we can do to help here SeanBatt. Very sorry you've had trouble... to echo Craig, their team has simplifying the install as a high priority. <3

**Comment by SeanBatt** [comment]
I think that you've only reiterated the very first command in the list of instructions. Executing it again in an elevated PowerShell hasn't helped me find wsl. I can see a wsl.exe in a WinSxS subdirectory, but it hasn't found its way into a directory on the path. I was going to reboot to see if that "fixed" things, as it too often does, but I note the command's  /norestart switch so surely one is not supposed to.
I'm perplexed by the suggestion that the instructions are too long, and simplifying them is a priority. I don't think they're too long, or too hard to follow, just that they don't work (for me and others) and so implicitly ask me to shave more yaks before I can get back to being productive.
Or not. Case closed: will not fix.

**Comment by craigloewen-msft** [comment]
Sean did you restart your machine afterwards? 
The error `wsl` is not recognized would indicate that `wsl.exe` is not on your machine or your PATH which means the `Windows Subsystem for Linux` optional feature is not enabled.
